base/compiler/ssair/verify.jl

```diff
@@ -28,6 +28,10 @@ function check_op(ir::IRCode, domtree::DomTree, @nospecialize(op), use_bb::Int,
                 error()
             end
         end
+    elseif isa(op, Union{OldSSAValue, NewSSAValue})
+        #@Base.show ir
+        @verify_error "Left over SSA marker"
+        error()
```

Comment:
need to fix these `error()` calls before release

Reply:
This is in the verifier.

Reply:
yes, that's why I said before release, not before merge